.travis.yml

```diff
@@ -11,17 +11,12 @@ before_install:
 install:
   - npm ci
```

Comment:
Once you've put `language: node_js` you don't need to specify `install`

Reply:
I think Nathan had a need of switching to npm ci (I saw the commit some days ago), so I guess we should keep it ? (comportement of npm ci is a little bit different from npm i)

Reply:
Removed, we found that `npm ci` is defaulting when package-lock is provided.